.github/workflows/build-rpm-pkg.sh

```diff
@@ -0,0 +1,28 @@
+PKG_ROOT=/tmp/avalanchego
+RPM_BASE_DIR=$PKG_ROOT/yum
+AVALANCHE_BUILD_BIN_DIR=$RPM_BASE_DIR/usr/local/bin
+AVALANCHE_LIB_DIR=$RPM_BASE_DIR/usr/local/lib/avalanchego
+
+mkdir -p $DEBIAN_BASE_DIR
+mkdir -p $AVALANCHE_BUILD_BIN_DIR
+mkdir -p $AVALANCHE_LIB_DIR
+
+OK=`cp ./build/avalanchego $AVALANCHE_BUILD_BIN_DIR`
+if [[ $OK -ne 0 ]]; then
+  exit $OK;
+fi
+OK=`cp ./build/plugins/evm $AVALANCHE_LIB_DIR`
+if [[ $OK -ne 0 ]]; then
+  exit $OK;
+fi
+
+echo "Build rpm package..."
+echo "Tag: $TAG"
+VER=$TAG
+if [[ $TAG =~ ^v ]]; then
```

Comment:
In `debian`, this is needed because the version tag inside the control file cannot contain a `v` (e.g. `v1.0.4`).

Does this apply for yum/rpm as well?

Reply:
Yum also can not use a v.  It has to just be a number.